infra/src/main/java/com/redhat/service/bridge/infra/models/dto/ConnectorStatus.java

```diff
@@ -4,9 +4,9 @@
 // https://issues.redhat.com/browse/MGDOBR-195
 public enum ConnectorStatus {
     REQUESTED,
```

Comment:
You might want to align to the new statuses (from MK) https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/333